crml/staking/src/rewards/mod.rs

```diff
@@ -87,19 +88,20 @@ decl_event!(
 decl_storage! {
 	trait Store for Module<T: Trait> as Rewards {
 		/// Inflation rate % to apply on reward payouts
-		pub BaseInflationRate get(fn inflation_rate): FixedU128 = FixedU128::saturating_from_rational(1u64, 100u64);
+		pub BaseInflationRate get(fn inflation_rate) config(): FixedU128 = FixedU128::saturating_from_rational(1u64, 100u64);
 		/// Development fund % take for reward payouts, parts-per-billion
 		pub DevelopmentFundTake get(fn development_fund_take) config(): Perbill;
 		/// Accumulated transaction fees for reward payout
 		pub TransactionFeePot get(fn transaction_fee_pot): BalanceOf<T>;
 		/// Historic accumulated transaction fees on reward payout
 		pub TransactionFeePotHistory get(fn transaction_fee_pot_history): VecDeque<BalanceOf<T>>;
-		/// Remaining reward amount of the eras which are not fully processed yet
-		pub QueuedEraRewards get(fn queued_era_rewards): VecDeque<BalanceOf<T>>;
 		/// Where the reward payment should be made. Keyed by stash.
+		// TODO: migrate to blake2 to prevent trie unbalancing
```

Comment:
As far as I know, Blake2 is necessary for what users have control over and can affect the storage. AccountId here is coming from User's account which are already issued by the system to be balanced.

Reply:
users can generate stash accounts as they like so the cost to unbalance this is fees + minimum bond, which makes it not urgent imo. `Bonded` is blake hashed upstream and we'll migrate to that in this PR also